packages/components/src/ui/context/context-connect.ts

```diff
@@ -16,42 +16,70 @@ import { CONNECT_STATIC_NAMESPACE } from './constants';
 import { getStyledClassNameFromKey } from './get-styled-class-name-from-key';
 import type { WordPressComponentFromProps } from '.';
 
+type AcceptsTwoArgs<
+	F extends ( ...args: any ) => any,
+	ErrorMessage = never
+> = Parameters< F >[ 'length' ] extends 2 ? {} : ErrorMessage;
+
 type ContextConnectOptions = {
-	/** Defaults to `false`. */
-	memo?: boolean;
```

Comment:
I removed this option because no components were using it 🤷  Let me know if anyone wants to keep it around.

Reply:
Nah, let's remove it and `memo` explicitly inside each component when needed